upgrade/install-3.0.php

```diff
@@ -79,5 +79,15 @@ function upgrade_module_3_0($object, $install = false)
 			) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 AUTO_INCREMENT=1');
 	}
 
+	//Add datas to paypal order
+	Db::getInstance()->Execute('
```

Comment:
Hi,
You need to create a new file named upgrade/install-3.8.3.php with a new function named upgrade_module_3_8_3($object, $install = false) with those data.

Best regards